### DISCUSSION TITLE
How to change the shape of points to circles

### DISCUSSION THREAD

**Discussion by WuKongBlog** [cnr-isti-vclab/meshlab#944] [Q&A]
- windows10 meshlab 2020.06
How to change the shape of points back to circle

![image](https://user-images.githubusercontent.com/68042053/111955360-5967f200-8b24-11eb-8a6a-bae90532421c.png)

**Comment by alemuntoni** [comment]
Set the "dot decorator", as done in this screenshot:
![image](https://user-images.githubusercontent.com/6656168/113701207-21da7780-96d8-11eb-866b-99ed1a0cb29b.png)

**Reply by WuKongBlog** [reply]
As shown in the screenshot, I've already clicked this button, but the point shape is still square. In addition, this is only a special case, which is displayed normally on other computers. I wonder if this is related to some configuration of the computer.

**Reply by alemuntoni** [reply]
Is this affecting also the 2020.12 version?

**Reply by WuKongBlog** [reply]
Yeah
![image](https://user-images.githubusercontent.com/68042053/113705144-a98fa880-970f-11eb-9ee3-89bce05da1d5.png)

**Reply by alemuntoni** [reply]
Oh ok, in that case I think that is an hardware-related issue... In my PC it works fine.

**Reply by WuKongBlog** [reply]
But the computer was normal before, so I think some files on the computer may have been modified.

**Reply by alemuntoni** [reply]
I don't know, it is hard to figure out what's going on here. 
I also see that the font in the bottom-right part of the window is different from the standard one, and this should not happen. 
I don't know what could cause this behaviour, I'm sorry.

**Reply by WuKongBlog** [reply]
Well, thank you very much. I guess I accidentally changed some of the configuration files of Windows10, but this issue does not affect other functions. I will try to fix it, and if necessary, I will consider reinstalling the system. Thank you again!
